scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -181,8 +181,28 @@ def configure_lnet(self):
     restart_services(self, ["lnet"])
 
 def configure_libfabric(self):
-    raise MotrError(errno.EINVAL, "libfabric not implemented\n")
+    try:
```

Comment:
Before accepted and merged to main, this modification must be tested thoroughly on Mini-provisioner platform.

Reply:
Ok. We will verify this before merging to main